### PR TITLE
:arrow_up: imp to importlib #892

### PR DESCRIPTION
**Pull request by tkoyama010**
:arrow_up: imp to importlib #892

### CODE REVIEW

**Comment by tkoyama010** [comment]
Can anybody solve this error? It seems the CI problem.
https://travis-ci.org/enthought/mayavi/jobs/648265707#L52

**Comment by mdickinson** [comment]
@tkoyama010 That error seems to be case of #861. I've restarted the failing Travis job to see if we can get it to pass.

File: mayavi/core/base.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from copy import deepcopy
 import os
 import logging
-import imp
+import importlib
```
Comment (mdickinson) [inline mayavi/core/base.py]: You also need to change the places where `imp` is used; I see one use in `__module_view_default`.

Comment (tkoyama010) [reply mayavi/core/base.py]: Thank you for your check. It's my fault.

**Comment by tkoyama010** [comment]
I also fixed the test source code :up:

**Comment by tkoyama010** [comment]
Ops. It fails in Python 2.7 :cry: Should I revert https://github.com/enthought/mayavi/pull/893/commits/7998d704cc96a60848195d002e6a391f63787f91
or ignore the error of Python2.7? #845

**Comment by tkoyama010** [comment]
I move importlib to reload_module in six :smile:

**Comment by prabhuramachandran** [comment]
@mdickinson -- is this OK to merge?  You have a review and I do not want to dismiss it.  Thanks.

**Comment by mdickinson** [comment]
@prabhuramachandran Sure. I still think there should be a test that exercises the code, but that's up to you. :-)

**Comment by prabhuramachandran** [comment]
Merging this now, I am addressing some tests (not the greatest test but it is something) in #913.

**Comment by tkoyama010** [comment]
Thanks.

**Pull request merged**